sceval_frontend/src/controllers/AppContext.ts

```diff
@@ -155,10 +126,9 @@ export class AppContext {
 
         if (loginInfo && loginInfo.user && loginInfo.authToken) {
           console.log('Validating saved login for', loginInfo.user);
-          ModeAPI.setAuthToken(loginInfo.authToken);
-          ModeAPI.request('GET', MODE_API_BASE_URL + '/users/' + loginInfo.user.id, {}, false)
-          .then(function(res: any) {
-            const user = res.data as User;
+          modeAPI.setAuthToken(loginInfo.authToken);
+          modeAPI.getUserInfo(loginInfo.user.id)
```

Comment:
These functions are asyncrhronous. So the order of processing functions is not guaranteed. Is it no problem?